crates/fuel-core/src/database/block.rs

```diff
@@ -114,9 +110,9 @@ impl Database {
         Ok(block.header().time().to_owned())
     }
 
-    pub fn get_block_id(&self, height: BlockHeight) -> StorageResult<Option<BlockId>> {
-        Database::get(self, &height.to_bytes()[..], Column::FuelBlockIds)
-            .map_err(Into::into)
+    pub fn get_block_id(&self, height: &BlockHeight) -> StorageResult<Option<BlockId>> {
+        let key = height.database_key();
+        Database::get(self, key.as_ref(), Column::FuelBlockIds).map_err(Into::into)
```

Comment:
```suggestion
        Database::get(self, height.database_key();.as_ref(), Column::FuelBlockIds).map_err(Into::into)
```
Is a bit more compact but just as readable imo

Reply:
Maybe you meant `height.database_key().as_ref()`?

Reply:
I did